relay-server/src/statsd.rs

```diff
@@ -481,25 +476,30 @@ pub enum RelayTimers {
     ///
     ///  - `message`: The type of message that was processed.
     ProcessMessageDuration,
-    /// Timing in milliseconds for handling a project cache message.
+    /// Timing in milliseconds for processing a task in the project cache service.
     ///
     /// This metric is tagged with:
-    ///  - `message`: The type of message that was processed.
-    ProjectCacheMessageDuration,
-    /// Timing in milliseconds for processing a message in the buffer service.
+    /// - `task`: The type of the task the project cache does.
```

Comment:
Why is this now `task`?

Reply:
The message is called `ProjectCacheTaskDuration`, also note this is a metric for the new service, the old one has new metrics.